dwave/plugins/sklearn/transformers.py

```diff
@@ -199,23 +200,30 @@ def correlation_cqm(
                 mode="w+",
                 shape=(X_copy.shape[1], X_copy.shape[1]),
                 )
-
+            # make the matrix that will hold the output correlations
+            target_correlations = np.memmap(
+                targetcor_out,
+                dtype=np.result_type(X, y),
+                mode="w+",
+                shape=(X_copy.shape[1], 1),
+                )
             # main calculation. It modifies X_copy in-place
             corrcoef(X_copy, out=correlations, rowvar=False, copy=False)
-
             # we don't care about the direction of correlation in terms of
             # the penalty/quality
             np.absolute(correlations, out=correlations)
-
+            # copying the correlations with the target column
+            target_correlations = correlations[:, -1].copy()
+            # multiplying all terms with (1 - alpha)
+            np.multiply(correlations, (1 - alpha), out=correlations)
```

Comment:
Rather than saving a separate `target_correlations` array, why not just
```suggestion
            np.multiply(correlations[:-1, :-1], (1 - alpha), out=correlations)
```